content/en/actions/app_builder/build.md

```diff
@@ -16,48 +16,102 @@ further_reading:
 
 You can create an app or edit existing apps from the [App Builder][1] page. The page lists information about existing apps, including the following:
 - Author
-- Status
-- Date that each app was last modified
+- Tags
+- Date that the app was last modified
 - Whether the app is published
 
 On the App Builder page, you can access and filter your apps. Hover over an app for options to edit, delete, view, or clone the app. You can also enable the **My apps** toggle to see only apps that you created:
 
 {{< img src="service_management/app_builder/app-builder-my-apps.png" alt="The App Builder page, filtered to show only 'My apps'" style="width:100%;" >}}
 
+## Start with onboarding
+
+When creating a new app, Datadog displays an onboarding modal with the following options: 
+- [Start with Blueprint][18]
+- [Start with Layout][19]
+- [Start with Data][20]
+- [Start from Scratch][21] 
+
+## See what's possible with App Playground
+
+If you're new to App Builder, you can use the App Playground to learn about its core concepts, including: 
+- Data fetching
+- Displaying query data in a table
+- Configuring a button to fire a query
+- Configuring the loading states for a component to reflect a query's loading state
+- Visualizing data in a graph
+
+To access the App Playground:
+1. From [App Builder][1], click **New App**. 
+1. Click **App Playground**.
+
 ## Create an app
 
 ### Build an app from a blueprint
 
-Blueprints are helpful starter apps that cover common use cases. They come loaded with demo data that you can use to familiarize yourself with the app. Blueprints also showcase best practices for setting up app functionality and visual presentation.
+Blueprints are helpful templates that cover common use cases. They come loaded with demo data that you can use to familiarize yourself with the app. Blueprints also showcase best practices for setting up app functionality and visual presentation.
 
-1. From [App Builder][1], click the [Blueprints][2] tab.
-1. Find the blueprint that you want to use and click **Preview**.
+1. From [App Builder][1], click the [Blueprints][2] tab. Alternatively, click **New App**, then **Start with Blueprint**.
+1. Find the blueprint that you want to use and click **View**.
```

Comment:
Is this under a feature flag? When I roll over a blueprint in demo, I still see Preview. Just wanted to check!